.github/workflows/unit-tests-ci.yml

```diff
@@ -2,6 +2,13 @@ name: Unit Tests Workflow
 
 on:
   workflow_call:
+    secrets:
+    github-token:
```

Comment:
Same here, bump these in.

Reply:
Just did. I don't understand why CI fails to start, I followed your docker example.

Reply:
> [Invalid workflow file: .github/workflows/continuous-integration.yml#L22](https://github.com/Seneca-CDOT/telescope/actions/runs/2150363645/workflow)
The workflow is not valid. .github/workflows/continuous-integration.yml (Line: 22, Col: 21): Invalid secret, github-token is not defined in the referenced workflow. .github/workflows/continuous-integration.yml (Line: 23, Col: 24): Invalid secret, turborepo-token is not defined in the referenced workflow.

From https://github.com/Seneca-CDOT/telescope/actions/runs/2150363645